class.jetpack.php

```diff
@@ -4429,7 +4443,8 @@ public function alert_auto_ssl_fail() {
 					$this.html( <?php echo json_encode( __( 'Checking', 'jetpack' ) ); ?> );
 					$( '#jetpack-recheck-ssl-output' ).html( '' );
 					e.preventDefault();
-					$.post( '/wp-json/jetpack/v4/recheck-ssl' )
+					var data = { action: 'jetpack-recheck-ssl', 'ajax-nonce': '<?php echo $ajax_nonce; ?>' };
```

Comment:
This is minor, and I'm not sure that we need to change this here. But, instead of `echo`ing inside of JS, we could localize the script. https://codex.wordpress.org/Function_Reference/wp_localize_script